var/spack/repos/builtin/packages/hdf/package.py

```diff
@@ -144,6 +144,22 @@ def configure_args(self):
             # We should not specify '--disable-hdf4-xdr' due to a bug in the
             # configure script.
             config_args.append('LIBS=%s' % self.spec['rpc'].libs.link_flags)
+
+        if '+fortran' in self.spec:
+            # https://github.com/Parallel-NetCDF/PnetCDF/issues/61
+            if self.spec.satisfies('%gcc@10:'):
```

Comment:
```suggestion
        # https://github.com/Parallel-NetCDF/PnetCDF/issues/61
        if self.spec.satisfies('+fortran %gcc@10:'):
```

Reply:
I think you are basically saying the flags can be set even if the fortran variant is off.  and exist even if the fortran variant isn't enabled. I am fine with that. It took two fixups to figure it out.

Reply:
The flake8 issues should be resolved.